exercise-src/react-native/11-navigation/01-problem/src/screens/Settings/Settings.test.tsx

```diff
@@ -14,26 +13,27 @@ jest.mock("../../design/theme", () => ({
   }),
 }))
 
-describe("Settings component", () => {
-  it("renders Settings Page", async () => {
+describe("Screens/Settings", () => {
+  it("renders", async () => {
     render(
       <NavigationContainer>
         <Settings />
       </NavigationContainer>,
     )
     expect(screen.getByText(/welcome/i)).toBeOnTheScreen()
   })
-  it("should switch to dark mode", () => {
+
+  it("switches to dark mode", () => {
     render(
       <NavigationContainer>
         <Settings />
       </NavigationContainer>,
     )
+
     const switchElement = screen.getByRole("switch")
     expect(switchElement.props.value).toBe(false)
-    fireEvent(switchElement, "onChange", {
-      nativeEvent: { value: true },
-    } as SwitchChangeEvent)
+
+    fireEvent(switchElement, "onChange", { nativeEvent: { value: true } })
```

Comment:
At some point I hope we can change these to `userEvent` instead.

Reply:
good catch. I was just removing the unnecessary cast. I've added it to the to-do list.